tmt/steps/execute/internal.py

```diff
@@ -260,22 +261,21 @@ def execute(
 
         # Create data directory, prepare test environment
         environment = self._test_environment(
-            test=test,
-            guest=guest,
+            invocation=invocation,
             extra_environment=extra_environment,
             logger=logger)
 
         # tmt wrapper filename *must* be "unique" - the plugin might be handling
         # the same `discover` phase for different guests at the same time, and
         # must keep them isolated. The wrapper script, while being prepared, is
         # a shared global state, and we must prevent race conditions.
-        test_wrapper_filename = f'{TEST_WRAPPER_FILENAME}.{guest.name}'
+        test_wrapper_filename = f'{TEST_WRAPPER_FILENAME}.{invocation.guest.name}'
```

Comment:
Same as above.

Reply:
Fixed by a rebase, `safe_name()` is supposed to be here.